index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
       
   <hr>
     </section> 
-  
+    <section class="works-desktop">
+      <div class="card-work1">
+        <div class="card-image">
+          <img src="./images/d1.jpg" alt="card-work">
+        </div>
+        <h2 class="h2">Tonic</h2>
+      <div class="left-block">
+          <div class="frame1">
+            <p>CONOPY</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">Back End Dev</p>
+            <img src="./images/Counter.jpg" alt="">
+            <p class="p1">2020</p>
+          </div>
+          
+            <p class="primary1">
+              A daily selection of privately<br />personalized reads; 
+            no accounts or<br /> sign-ups required.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action">
+            <a href="#"><p>See Project</p></a>
+          </div>
+          </div>
+      </div>  
+                   <div class="card-work2">
+                <div class="card-image2">
+                  <img src="./images/d2.png" alt="card-work">
+                </div>
+                <div class="left1-block">
+                  <h2 class="h2">
+                    Multi-Post
+                    Stories
+                  </h2>
+                  <div class="frame2">
+                    <p>FACEBOOK</p>
+                      <img src="./images/Counter.png" alt="">
+                    <p class="p1">FULL STACK DEV</p>
+                      <img src="./images/Counter.png" alt="">
+                    <p class="p1">2015</p>
+                  </div>
+                  <p class="primary2">
+                    Experimental content creation feature that allows users to<br /> add to an existing story over the course of a day without<br /> spamming their friends.
+                  </p>
+                  <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
```

Comment:
- [ ] I would suggest you, try to stick as much as possible to Figma design. Your design is not the same as Figma's design.
- [ ] The page is divided half of it contains the project's picture and the other half contains the project's description. So kindly fix it.
- [ ] Your `a` tag for `HTML`, `Ruby`, `css`, `javascript` is not matching the figma design kindly fix it.
### Your design
![middle](https://user-images.githubusercontent.com/81727988/167232696-0aa2b14b-926c-4d6f-9744-56dae40075d9.png)
### Figma design
![figma22](https://user-images.githubusercontent.com/81727988/167232703-5d542587-b109-47b5-a772-e1dcc39594ed.png)

Reply:
- [ ] [REQUESTED] as requested by the previous reviewer  Your a tag for HTML, Ruby, css, javascript is not matching the figma design kindly fix it.